renderer/reducers/invoice.js

```diff
@@ -33,6 +32,7 @@ export const SEND_INVOICE = 'SEND_INVOICE'
 export const INVOICE_SUCCESSFUL = 'INVOICE_SUCCESSFUL'
 export const INVOICE_FAILED = 'INVOICE_FAILED'
 export const UPDATE_INVOICE = 'UPDATE_INVOICE'
+export const SET_OFFSET = 'SET_OFFSET'
```

Comment:
Need reducer snapshot test for this new action too

Reply:
This is a leftover and was removed in the update